### DISCUSSION TITLE
Too much of empty space between webpart title and first item in search results layout

### DISCUSSION THREAD

**Discussion by yvsuhas** [microsoft-search/pnp-modern-search#3196] [Q&A]
**Version used**
Ex: 4.8

**Describe the bug**
There is more empty space between the webpart title and the first item in the chosen layout in the search results webpart.  The PnP Search webparts look little bit off from the other OOTB webparts as this space is not an issue with them.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Go to a SharePoint site page.
2. Add a PnP Search results webpart.
3. Add a title to the webpart.
4. Enter a search query to get people results.
5. You can see there is too much space between the title and the first item.

**Expected behavior**
Empty space between the webpart and the first item in the search results should be trimmed to match the OOTB webpart layout.

**Screenshots**
![PnP Search Layout Issue](https://github.com/microsoft-search/pnp-modern-search/assets/19211871/266e12c8-8c44-48bf-958f-2f9cb8953977)

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, Edge]

**Additional context**
N/A

**Comment by kasperbolarsen** [comment]
Hi
It looks like space of the space is reserved for the hits counter and the space is not reclaimed when the "Show results count" option is disabled.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/cf05255b-1955-49c4-a845-496ec5be3712)

**Comment by kasperbolarsen** [comment]
You should be able to change the height of that space using css in the people layout if you are familiar with css.
